src/components/ContactsList/ContactsTable.tsx/StatusBadge.tsx

```diff
@@ -0,0 +1,19 @@
+import { ContactAwaitingAcceptance, ContactConfirmed, ContactNotConfirmed, ContactStatus } from '@/api/types/contact'
+import { FC } from 'react'
+import { Chip, ChipProps } from '@mui/material'
+
+type StatusBadgeProps = {
+  status: ContactStatus
+}
+
+export const StatusBadge: FC<StatusBadgeProps> = ({ status }) => {
+  const { label, color } = dictionary[status]
+
+  return <Chip size="small" label={label} color={color} />
+}
+
+const dictionary: Record<ContactStatus, { label: string; color: ChipProps['color'] }> = {
```

Comment:
`contactStatuses` could be a better name for dictionary. WDYT ?

Reply:
It's local, not-exported map and it's in the file which refers to the "Status" keyword. 
Hovewer, I will change as you're suggesting 👍